packages/runtime/container-runtime/src/test/gc/gcTelemetry.spec.ts

```diff
@@ -514,28 +501,26 @@ describe("GC Telemetry Tracker", () => {
 
 				// Update all nodes again. There shouldn't be any more events since for each node the event is only once.
 				mockNodeChanges(nodes);
-				await simulateGCToTriggerEvents(isSummarizerClient);
+
 				validateNoEvents();
 			});
 
 			// This test is only relevant for summarizer client because it does not log changed events if the node is revived.
 			if (isSummarizerClient) {
-				it("generates only revived event in summarizer when a node is updated and revived", async () => {
+				it("generates only revived event in summarizer when a node is revived then updated", async () => {
 					// Mark node 2 as unreferenced.
 					markNodesUnreferenced([nodes[2]]);
 
 					// Advance the clock just before the timeout and validate no unexpected events are logged.
 					clock.tick(timeout - 1);
 					mockNodeChanges(nodes);
-					await simulateGCToTriggerEvents(isSummarizerClient);
 
 					validateNoEvents();
 
 					// Expire the timeout and validate that only revived event is generated for node 2.
 					clock.tick(1);
+					mockNodeRevival(nodes[1], nodes[2]);
 					mockNodeChanges([nodes[2]]);
```

Comment:
Switching order here so that the revival happens before the change, since we're logging directly instead of queuing?